[2024-03-04T17:48:47.912+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.get_data_task manual__2024-03-04T17:48:44.892614+00:00 [queued]>
[2024-03-04T17:48:47.920+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.get_data_task manual__2024-03-04T17:48:44.892614+00:00 [queued]>
[2024-03-04T17:48:47.920+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T17:48:47.920+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-03-04T17:48:47.920+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T17:48:47.928+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): get_data_task> on 2024-03-04 17:48:44.892614+00:00
[2024-03-04T17:48:47.932+0000] {standard_task_runner.py:55} INFO - Started process 164 to run task
[2024-03-04T17:48:47.933+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gera_dados_turbina', 'get_data_task', 'manual__2024-03-04T17:48:44.892614+00:00', '--job-id', '3398', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpt7ee9wu2']
[2024-03-04T17:48:47.934+0000] {standard_task_runner.py:83} INFO - Job 3398: Subtask get_data_task
[2024-03-04T17:48:47.970+0000] {task_command.py:388} INFO - Running <TaskInstance: gera_dados_turbina.get_data_task manual__2024-03-04T17:48:44.892614+00:00 [running]> on host f002d2097047
[2024-03-04T17:48:48.011+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=adrianolima095@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gera_dados_turbina
AIRFLOW_CTX_TASK_ID=get_data_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-04T17:48:44.892614+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-04T17:48:44.892614+00:00
[2024-03-04T17:48:48.044+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-03-04T17:48:48.048+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=gera_dados_turbina, task_id=get_data_task, execution_date=20240304T174844, start_date=20240304T174847, end_date=20240304T174848
[2024-03-04T17:48:48.065+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-03-04T17:48:48.082+0000] {taskinstance.py:2578} INFO - 3 downstream tasks scheduled from follow-on schedule check
